cg/cli/deliver/base.py

```diff
@@ -133,6 +133,54 @@ def deliver_ticket(
     )
 
 
+@deliver.command(
+    name="sample",
+    help="Deliver fastq or bam files for a sample to the customer inbox on the HPC "
+    "and start an Rsync job to clinical-delivery. "
+    "NOTE: the dry-run flag will copy files to the customer inbox on Hasta, "
+    "but will not perform the Rsync job.",
+)
+@click.pass_obj
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="The case the sample is on.",
+)
+@click.option("-s", "--sample-id", required=True, help="Deliver the files for a specific sample.")
+@click.option(
+    "-d",
+    "--delivery-type",
+    required=True,
+    help="The delivery type to use.",
+    type=click.Choice(choices=["fastq", "bam"]),
+)
```

Comment:
Should we not get this from the case?

Reply:
No, if the case is not deliverytype fastq or bam you couldn't deliver raw files for it that way.

Reply:
And we want this functionality to work for cases which does not have those delivery types? If so, please resolve

Reply:
We want to be able to deliver raw-data for single samples when we want to even if the original case has only "analysis" as delivery type.

Reply:
I also feel that this could be extracted, it is not as a sample could be delivered in any of the two ways. Why not getting this from the application version in the sample? Would we need the PR of the new table for this? @islean

Reply:
The application version has no relation to this part of the code.

If a customer requests delivery of fastq files for a sample tjat was initially ordered as "analysis" you will need to provide the requested delivery type in order to override it. This is a feature requested by prod.

Reply:
My concerns are mainly centered around what happens if the provided case can be misaligned with the options provided by the production member in any way